src/lucky_migrator/alter_table_statement.cr

```diff
@@ -81,6 +92,14 @@ class LuckyMigrator::AlterTableStatement
     end
   end
 
+  def add_fill_existing_with_statements(column : Symbol, value : ColumnDefaultType)
+    val = default_value(value.class, value).gsub(" DEFAULT ", "")
```

Comment:
I think you're right that we should have some other method that we can use for this. Maybe `value_to_string` that this method and `default_value` can use. Would that work?

Reply:
Great, I'll put that in `ColumnDefaultHelpers`

Reply:
K its up